### PR TITLE
[coinmate] metadata

### PR DESCRIPTION
**Pull request by npomfret**

### CODE REVIEW

File: js/coinmate.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = class coinmate extends Exchange {
             },
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/27811229-c1efb510-606c-11e7-9a36-84ba2ce412d8.jpg',
-                'api': 'https://coinmate.io/api',
+                'api': 'https://coinmate.docs.apiary.io',
```
Comment (kroitor) [inline js/coinmate.js:30]: I think this is a wrong key to edit the doc url )

Comment (kroitor) [reply js/coinmate.js:30]: See line 34 below.

Comment (npomfret) [reply js/coinmate.js:30]: https://coinmate.io/api just 404s

Comment (npomfret) [reply js/coinmate.js:30]: Wrong key?

Comment (npomfret) [reply js/coinmate.js:30]: Oh i see, silly me. thanks

**Pull request closed without merging**